### ISSUE TITLE
Introduce configuration

### ISSUE THREAD

**Issue by dk1844** [AbsaOSS/login-gateway#5]
Introduce a configuration setup, most probably implemented by TypeSafe config.

Atum-Service project can be used for ideas, some version of TypeSafe config exists there and should be compatible.

## Design hints
Here, we are looking to provide some configuration defaults and external overrides per deployment needs.
This can perhaps be a form of `reference.conf` (internal defaults)+`application.conf` (externally provided) or also overriding individual fields external `-Dconfig.entry=value`.

Inspiration can be taken from https://github.com/AbsaOSS/atum-service -- it has a similar setup.

### List of users
~Optionally, configuration for defined set of users may be sketched out. (This especially makes sense when implementing after #7)~ moved to a separate issue #21
